examples/apps/src/ble_bas_peripheral.rs

```diff
@@ -74,7 +74,11 @@ where
                     // then return to advertising state.
                     select(connection_task, counter_task).await;
                 }
-                Err(_) => info!("[adv] error"),
+                Err(e) => {
+                    #[cfg(feature = "defmt")]
+                    let e = defmt::Debug2Format(&e);
```

Comment:
Would this work even without defmt?

Reply:
I'm not sure what you mean, because it does compile on the non-defmt examples

Reply:
Ok, so `defmt::Debug2Format` fixes the value for panic formatter.. Apologies for the noise.